.gitlab-ci.yml

```diff
@@ -67,3 +67,9 @@ clang-6.0:
         CONAN_CLANG_VERSIONS: "6.0"
     <<: *build-template
 
+clang-7:
+    image: conanio/conanclang7
```

Comment:
@offa I missed this in my review, and the build failed as expected: https://gitlab.com/bryceschober/conan-cpputest/-/jobs/113243435

Reply:
I should have copy & pasted this …
I'll fix this (again).

Reply:
PR is open #16